frontend/src/store.ts

```diff
@@ -4769,11 +4839,29 @@ export function createStore(web3: Web3) {
         const { Treasury } = state.contracts();
         if(!Treasury || !state.defaultAccount) return;
 
-        const partnerProject = partnerProjectFromContract(
-          await Treasury.methods.getPartnerProject(id).call(defaultCallOptions(state))
-        );
-
-        commit('updatePartnerProjectsState', { partnerProjectId: id, partnerProject });
+        const partnerProjectRaw = await Treasury.methods.partneredProjects(id).call(defaultCallOptions(state));
+        const tokensClaimed = await Treasury.methods.tokensClaimed(id).call(defaultCallOptions(state));
+        const logo = await Treasury.methods.projectLogo(id).call(defaultCallOptions(state));
+        const details = await Treasury.methods.projectDetails(id).call(defaultCallOptions(state));
+        const website = await Treasury.methods.projectWebsite(id).call(defaultCallOptions(state));
+        const note = await Treasury.methods.projectNote(id).call(defaultCallOptions(state));
```

Comment:
I'd add a view that returns all these details in one call, I think it will be better performance wise, especially with Treasury page invoking this function every 3 seconds.

Reply:
Done